src/render/NewGameScreen.java

```diff
@@ -64,12 +69,12 @@ public void blit(SpriteBatch spriteBatch, boolean focused) {
     public NewGameScreen(Game game) {
         BitmapFont font = Fonts.getButtonFont();
 
-        left.add(new ToggleButton(0, 0));
-        right.add(new ToggleButton(COLOR_WIDTH, 0));
-        left.add(new ToggleButton(0, COLOR_HEIGHT));
-        right.add(new ToggleButton(COLOR_WIDTH, COLOR_HEIGHT));
+        left.add(color1Button = new ToggleButton(0, 0));
+        right.add(color2Button = new ToggleButton(COLOR_WIDTH, 0));
+        left.add(color3Button = new ToggleButton(0, COLOR_HEIGHT));
+        right.add(color4Button = new ToggleButton(COLOR_WIDTH, COLOR_HEIGHT));
         left.add(new TextButton(font, "Exit", () -> game.stop()));
-        right.add(new TextButton(font, "Ready", () -> game.enterWorld()));
+        right.add(new TextButton(font, "Ready", () -> game.enterWorld(atLeastOneSelected)));
```

Comment:
It may help to define a new private function in this class, maybe call it tryStartGame, and put the "check if valid and print" logic there, then only enter the world if the requirements are met. You can link the button to the new function by changing () -> game.enterWorld() to () -> this.tryStartGame(game)

Reply:
OK, I'll give that a try.